libctru/include/3ds/services/cfgu.h

```diff
@@ -170,3 +176,9 @@ Result CFGI_VerifySigLocalFriendCodeSeed(void);
  * @brief Verifies the RSA signature for the SecureInfo data already stored in memory.
  */
 Result CFGI_VerifySigSecureInfo(void);
+
+/**
+ * @brief Gets the system's serial number.
+ * @param serial Pointer to output the serial to.
```

Comment:
Could you add a small comment saying how many bytes the buffer is required to have?